tests/baselines/reference/asyncFunctionDeclaration15_es6.errors.txt

```diff
@@ -1,12 +1,8 @@
-tests/cases/conformance/async/es6/functionDeclarations/asyncFunctionDeclaration15_es6.ts(6,16): error TS1055: Type '{}' is not a valid async function return type.

```

Comment:
Should this error be removed? it seems that this error `The return type of an async function or method must be the global Promise<T>.` replaces it